qiskit/circuit/library/standard_gates/swap.py

```diff
@@ -106,7 +106,7 @@ def __array__(self, dtype=None):
 
 
 class CSwapGate(ControlledGate):
-    r"""Controlled-X gate.
+    r"""Controlled swap gate.
```

Comment:
For consistency with other documentation can you write this as "Controlled-SWAP gate, also known as the Fredkin gate."?

Reply:
Thanks!